docs/tutorials/qml/classification.md

```diff
@@ -0,0 +1,118 @@
+In this tutorial we will show how to use Qadence to solve a basic classification task using a hybrid quantum-classical model composed of a QNN and classical layers.
+
+## Dataset
+
+We will use the Iris dataset separated into training and testing sets. A StandardScaler is used to normalize the data.
+
+```python exec="on" source="material-block" session="classification"
+
+import random
+
+import torch
+import torch.nn as nn
+from sklearn.datasets import load_iris
+from sklearn.preprocessing import StandardScaler
+from sklearn.model_selection import train_test_split
+from torch import Tensor
+from torch.utils.data import DataLoader, Dataset
+
+from qadence import QNN, RX, FeatureParameter, QuantumCircuit, Z, chain, hea, kron
+from qadence.ml_tools import TrainConfig, Trainer
+
+class IrisDataset(Dataset):
+    """The Iris dataset split into a training set and a test set.
+
+    A StandardScaler is applied prior to applying models.
+    """
+
+    def __init__(self):
+        X, y = load_iris(return_X_y=True)
+        X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.33, random_state=42)
+
+        scaler = StandardScaler()
+        scaler.fit(X_train)
+        self.scaler = scaler
+        self.X = torch.tensor(scaler.transform(X_train), requires_grad=False)
+        self.y = torch.tensor(y_train, requires_grad=False)
+
+        self.X_test = torch.tensor(scaler.transform(X_test), requires_grad=False)
+        self.y_test = torch.tensor(y_test, requires_grad=False)
+
+    def __getitem__(self, index) -> tuple[Tensor, Tensor]:
+        return self.X[index], self.y[index]
+
+    def __len__(self) -> int:
+        return len(self.y)
+
+n_features = 4  # sepal length, sepal width, petal length, petal width
+n_layers = 3
+n_neurons_final_linear_layer = 3
+n_epochs = 1000
+lr = 1e-1
+dataset = IrisDataset()
+
+dataloader = DataLoader(dataset, batch_size=20, shuffle=True)
+
+```
+
+
+## Hybrid QNN
+
+We set up the QNN part composed of multiple feature map layers, each followed by a variational layer.
+The output will be the expectation value wrt a Z observable on qubit 0
+Then we add a simple linear layer serving as as classification head. Note softmax is not applied with the cross-entropy loss.
```

Comment:
Not sure I totally get that last sentence. Could you elaborate a bit more please ?